plugins/reminder.rb

```diff
@@ -0,0 +1,35 @@
+#  -*- coding: utf-8 -*-
+#  reminder.rb
+#  Author: slackR | slackErEhth77
+#  ------------------------
+#  A Cinch plugin to set reminder  for a given time.
+#  ------------------------
+#  This code is licensed by slackErEhth77 under the MIT License.
+#  http://opensource.org/licenses/MIT
+
+require_relative 'yossarian_plugin'
+
+	class Reminder < YossarianPlugin
+		include Cinch::Plugin
+		
```

Comment:
add `use_blacklist` below this include so that the plugin responds to blacklist blocks